### PR TITLE
SE-20844 - Implement sampling on a client analytics

### PR DESCRIPTION
**Pull request by SoHotSoup**

### CODE REVIEW

**Comment by SoHotSoup** [comment]
@Vunovati could you do the same for Android?

**Comment by Vunovati** [comment]
Sure. I will create a separate task.

File: ios/UniversalAnalyticsPlugin.m [hunk 1]
```diff
@@ -374,4 +374,16 @@ - (void) addTransactionItem: (CDVInvokedUrlCommand*)command
...
+- (void) setSampling: (CDVInvokedUrlCommand*)command
+{
+    NSNumber *sampling = [command.arguments objectAtIndex:0];
+    NSString *trackerId = [command.arguments objectAtIndex:1];
+    NSString *samplingValue = [NSString stringWithFormat:@"%.1g", [sampling doubleValue]];
+    
+    id<GAITracker> tracker = [self getTrackerWithId:trackerId];
+    
+    [tracker set:kGAISampleRate value:samplingValue];
+
```
Comment (markomatic) [inline ios/UniversalAnalyticsPlugin.m:386]:
```
[self.commandDelegate sendPluginResult:[CDVPluginResult resultWithStatus:CDVCommandStatus_OK]
                                                   callbackId:command.callbackId];
```

And remove empty line at bottom.

**Comment by markomatic** [comment]
Update www/analytics.js with new function before merging.

**Pull request merged**